4.x/typescript/arcgis-js-api.d.ts

```diff
@@ -3037,10 +3037,10 @@ declare namespace __esri {
   export const SimpleMarkerSymbol: SimpleMarkerSymbolConstructor;
 
   interface SimpleMarkerSymbolProperties extends MarkerSymbolProperties {
-    color?: Color;
+    color?: Color | string;
     outline?: SimpleLineSymbolProperties;
     path?: string;
-    size?: number;
+    size?: number | string;
```

Comment:
The types of these properties need to match the type that's returned when the property is read to prevent the need for type guards in the code.